capa/features/extractors/binja/helpers.py

```diff
@@ -51,3 +51,21 @@ def unmangle_c_name(name: str) -> str:
             return match.group(1)
 
     return name
+
+
+def read_c_string(bv: BinaryView, offset: int, max_len: int) -> str:
+    s = ""
+    count = 0
+    while count < max_len:
+        try:
+            c = bv.read(offset + count, 1)[0]
+        except Exception:
+            break
+
+        if c == 0:
+            break
+
+        s += chr(c)
+        count += 1
+
+    return s
```

Comment:
```suggestion
    s = []
    while len(s) < max_len:
        try:
            c = bv.read(offset + count, 1)[0]
        except Exception:
            break

        if c == 0:
            break

        s.append(chr(c))

    return "".join(s)
```

using `s += c` in a loop like this is O(n**2), which may not matter for small `n`, but seems appropriate to avoid this style in general.

Reply:
> using `s += c` in a loop like this is O(n**2), which may not matter for small `n`, but seems appropriate to avoid this style in general.

I have fixed the code in the latest push